src/Session/FileSessionStore.php

```diff
@@ -322,7 +324,7 @@ public function destroy(int $expiryTime, string $id)
 		}
 
 		// file still exists, delete it
-		if (@unlink($path) !== true) {
+		if (F::unlink($path) !== true) {
```

Comment:
If we don't swallow the warning in `F::unlink()` (see comment above), we need to do it here and keep the `@` error control operator.